src/components/shop/confirm/confirm-resize.js

```diff
@@ -3,8 +3,7 @@
  * @author BBC Children's D+E
  * @license Apache-2.0
  */
-import { getInnerRectBounds, getSafeArea } from "../shop-layout.js";
```

Comment:
I wonder if we should remove `getSafeArea` from `shop-layout.js`, should always be able to use `scene.layout.getSafeArea`

Reply:
Yeah I've been doing that where I see it. Just obscures what's going on.